doc/releases/changelog-0.43.0.md

```diff
@@ -909,6 +909,9 @@
 
 <h3>Breaking changes 💔</h3>
 
+* Pin `autoray` to `v0.8.0` for PennyLane `v0.43.0` release to prevent potential bug fix releases.
+  [(#8412)](https://github.com/PennyLaneAI/pennylane/pull/8412)
```

Comment:
```suggestion
* `autoray == 0.8.0` is proactively pinned for PennyLane v0.43.0 to prevent potential bug fix releases.
  [(#8412)](https://github.com/PennyLaneAI/pennylane/pull/8412)
```

Reply:
NB: Due to this branch not being able to be merged into the correct branch that includes changelog changes, we will add this entry manually to https://github.com/PennyLaneAI/pennylane/pull/8398